rustup/Dockerfile

```
@@ -0,0 +1,24 @@
+FROM rust:latest AS builder
+
```

如果想的话，把自己添加为 maintainer:

```suggestion
LABEL maintainer="Your name <example AT ustclug.org>"
```